### DISCUSSION TITLE
S7 Core Library Update

### DISCUSSION THREAD

**Discussion by lupaulus** [gijzelaerr/python-snap7#489] [General]
I would like to ask, if you know when some update of the core library snap7 are planned because the project seems standing ?

**Comment by spreeker** [comment]
Hi Paulus,

What are you missing? It is just a wrapper around snap7.

Still works as intended. Need some new support?

**Reply by lupaulus** [reply]
At the moment, my Test Bench with a S7-1518F was working well but in production I will have an S7-417.

When I see this [issue](https://github.com/gijzelaerr/python-snap7/discussions/470), I'm afraid of doing a complete factory stop...

**Comment by spreeker** [comment]
As niketly mentioned in the comments on that issue. Python library is not
the problem. If the underlying snap7 library is up to date you should be
fine. Just run some test to be sure as you should.

Stephan.

**Comment by gijzelaerr** [comment]
Closing as resolved. Python-snap7 3.0 is now a pure Python implementation and no longer depends on the C snap7 library at all.
